packages/lexical-list/src/LexicalListNode.ts

```diff
@@ -49,6 +50,26 @@ export type SerializedListNode = Spread<
 
 export type ListType = 'number' | 'bullet' | 'check';
 
+export type ListStyleType =
+  | null // null means default
```

Comment:
```suggestion
  | undefined // default
```

`undefined` is probably an easier default to manage in this case